data/src/main/java/cmm/apps/esmorga/data/event/EventRepositoryImpl.kt

```diff
@@ -28,9 +28,18 @@ class EventRepositoryImpl(private val localUserDs: UserDatasource, private val l
 
     override suspend fun joinEvent(eventId: String) {
         remoteEventDs.joinEvent(eventId)
+        updateCacheEvents(eventId, true)
+    }
+
+    override suspend fun leaveEvent(eventId: String) {
+        remoteEventDs.leaveEvent(eventId)
+        updateCacheEvents(eventId, false)
```

Comment:
instead of calling this, I think it would be cleaner to implement `leaveEvent` in the local DataSource and call it here too.